core/controllers/profile.py

```diff
@@ -253,7 +253,7 @@ def put(self) -> None:
         email = self.normalized_payload['email']
         name = self.normalized_payload['name']
         tag = self.normalized_payload['tag']
-        status = user_services.add_user_to_mailing_list(email, name, tag)
+        status = user_services.add_user_to_mailing_list(email, tag, name)
```

Comment:
This should be add_uiser_to_mailing_list(email, tag, name=name).

(For default args, always include the arg name at the callsite. Please fix anywhere else that's appropriate.)

Reply:
Hey, I'm having a bit of a problem. After changing this with the keyword argument `name=name` I had to also change the test file for the `test_put_function` but don't know why I am not able to correctly update the test file and ended up failing `backend_tests`. :(

So, I decided to not push those changes to the test file. Any help would be highly appreciated. Thanks

Reply:
I don't know either because I don't have enough information. I suggest that you create a [debugging doc](https://docs.google.com/document/d/1qRbvKjJ0A7NPVK8g6XJNISMx_6BuepoCL7F2eIfrGqM/edit) and share it, with information about what you tried and why.

Reply:
I have been trying to fix this issue for a few days but since it is a keyword argument. the `swap` function had to be changed to also accept the keyword argument, I modified it as well. Don't know if because of this change the app is breaking somehow and the backend test is failing.


https://github.com/oppia/oppia/blob/622e833ee29cbb35e4e3e28b484503fc7b6f0ca4/core/controllers/profile_test.py#L988-L989

I was wondering, do we need to handle it this seriously for such non non-breaking change? I know this sounds bad but.. :(

Reply:
I apologize for the late response to the review. :( I actually made a silly mistake of not including `**kwargs` in the test code `test_put_function`. Now, everything should work as expected and the test should pass.  PTAL